langs/he.json

```diff
@@ -26,107 +44,146 @@
+    "mh": "מרשליזית",
+    "mi": "מאורי",
     "mk": "מקדונית",
-    "ml": "מליאלאם",
+    "ml": "מליאלם",
     "mn": "מונגולית",
-    "mr": "מראטהית",
+    "mr": "מראטהי",
     "ms": "מלאית",
     "mt": "מלטית",
     "my": "בורמזית",
-    "nb": "נורווגית ספרותית",
-    "ne": "נפאלית",
-    "nl": "הולנדית",
-    "nn": "נורווגית חדשה",
-    "no": "נורווגית",
-    "ny": "ניאנג׳ה",
-    "oc": "אוקסיטנית",
-    "om": "אורומו",
-    "or": "אורייה",
-    "pa": "פנג׳אבי",
-    "pl": "פולנית",
-    "ps": "פאשטו",
-    "pt": "פורטוגזית",
-    "qu": "קצ׳ואה",
-    "rm": "רומאנש",
-    "rn": "קירונדי",
-    "ro": "רומנית",
-    "ru": "רוסית",
-    "rw": "קנירואנדית",
-    "sd": "סינדהית",
-    "si": "סינהלה",
-    "sk": "סלובקית",
-    "sl": "סלובנית",
-    "sm": "סמואית",
-    "sn": "שונה",
-    "so": "סומלית",
-    "sq": "אלבנית",
-    "sr": "סרבית",
-    "st": "סותו דרומית",
-    "su": "סונדנזית",
-    "sv": "שוודית",
-    "sw": "סווהילי",
-    "ta": "טמילית",
-    "te": "טלוגו",
-    "tg": "טג׳יקית",
-    "th": "תאית",
-    "ti": "תיגרינית",
-    "tk": "טורקמנית",
-    "tl": "פיליפינית",
-    "tn": "סוואנה",
-    "to": "טונגאית",
-    "tr": "טורקית",
-    "tt": "טטרית",
-    "tw": "אקאן",
-    "ug": "אויגורית",
-    "uk": "אוקראינית",
-    "ur": "אורדו",
-    "uz": "אוזבקית",
-    "vi": "וייטנאמית",
-    "wa": "ולונית",
-    "wo": "וולוף",
-    "xh": "קוסה",
-    "yi": "יידיש",
-    "yo": "יורובה",
-    "zh": "סינית",
-    "zu": "זולו"
+    "na": "נאורית",
+    "nb": "Νορβηγικά Μποκμάλ",
+    "nd": "Βόρεια Ντεμπέλε",
+    "ne": "Νεπαλικά",
+    "ng": "Ντόνγκα",
+    "nl": "Ολλανδικά",
+    "nn": "Νορβηγικά Νινόρσκ",
+    "no": "Νορβηγικά",
+    "nr": "Ντεμπέλε Νδέμπελε",
+    "nv": "Ναβάχο",
+    "ny": "Νιάντζα",
+    "oc": "Οκσιτανικά",
+    "oj": "Οζιμπουά",
+    "om": "Ορόμο",
+    "or": "Ορίγια",
+    "os": "Οσετικά",
+    "pa": "Παντζαπικά",
+    "pi": "Πάλι",
+    "pl": "Πολωνικά",
+    "ps": "Πάστο",
+    "pt": "Πορτογαλικά",
+    "qu": "Κετσούα",
+    "rm": "Ρομανικά",
+    "rn": "Ρούντι",
+    "ro": "Ρουμανικά",
+    "ru": "Ρωσικά",
+    "rw": "Κινιαρβάντα",
+    "sa": "Σανσκριτικά",
+    "sc": "Σαρδηνικά",
+    "sd": "Σίντι",
+    "se": "Βόρεια Σάμι",
+    "sg": "Σάνγκο",
+    "si": "Σινχαλεζικά",
+    "sk": "Σλοβακικά",
+    "sl": "Σλοβενικά",
+    "sm": "Σαμοανά",
+    "sn": "Σχόνα",
+    "so": "Σομαλικά",
+    "sq": "Αλβανικά",
+    "sr": "Σερβικά",
+    "ss": "Σουάτι",
+    "st": "Νότια Σόθο",
+    "su": "Σουδανικά",
+    "sv": "Σουηδικά",
+    "sw": "Σουαχίλι",
+    "ta": "Ταμίλ",
+    "te": "Τελούγκου",
+    "tg": "Τατζίκ",
+    "th": "Ταϊλανδέζικα",
+    "ti": "Τιγρίνυα",
+    "tk": "Turkmen",
+    "tl": "Tagalog",
+    "tn": "Tswana",
+    "to": "Tongan",
+    "tr": "Turkish",
+    "ts": "Tsonga",
+    "tt": "Tatar",
+    "tw": "Twi",
+    "ty": "Tahitian",
+    "ug": "Uyghur",
+    "uk": "Ukrainian",
+    "ur": "Urdu",
+    "uz": "Uzbek",
+    "ve": "Venda",
+    "vi": "Vietnamese",
+    "vo": "Volapük",
+    "wa": "Walloon",
```

Comment:
This one for example was having correct hebrew typography before the changes 🤔

Reply:
I saw 2 problems here:

1. The 2nd and 3rd parts had wrongly Greek and English names – that was an accident during copypasting from ChatGPT. Sorry for that, fixed it.
2. Chat GPT seems to have changed existing translations. I have thus sent them through ChatGPT again, with the command *fix those entries, use correct Hebrew orthography*. Not sure if it helped, I don't speak Hebrew. I understand if you don't want to merge this part. I could also create a commit with no changes to existing translations.

Reply:
I won't merge chatGPT based translations. those seem to be to often wrong